presto-memory/src/test/java/io/prestosql/plugin/memory/TestMemorySmoke.java

```diff
@@ -260,16 +260,16 @@ public void testSemiJoinDynamicFilteringBlockProbeSide()
                         "WHERE t.partkey IN (SELECT p.partkey FROM part p WHERE p.comment = 'onic deposits')",
                 withBroadcastJoinNonReordering(),
                 1,
-                ImmutableSet.of(1, ORDERS_COUNT, PART_COUNT));
+                ImmutableList.of(1, ORDERS_COUNT, PART_COUNT));
     }
 
-    private void assertDynamicFiltering(@Language("SQL") String selectQuery, Session session, int expectedRowCount, Set<Integer> expectedOperatorRowsRead)
+    private void assertDynamicFiltering(@Language("SQL") String selectQuery, Session session, int expectedRowCount, List<Integer> expectedOperatorRowsRead)
     {
         DistributedQueryRunner runner = (DistributedQueryRunner) getQueryRunner();
         ResultWithQueryId<MaterializedResult> result = runner.executeWithQueryId(session, selectQuery);
 
         assertEquals(result.getResult().getRowCount(), expectedRowCount);
-        assertEquals(getOperatorRowsRead(runner, result.getQueryId()), expectedOperatorRowsRead);
+        assertEqualsNoOrder(getOperatorRowsRead(runner, result.getQueryId()).toArray(), expectedOperatorRowsRead.toArray());
```

Comment:
why no order? Why not to compare lists directly?

Reply:
Sounds good - fixed 9626884a13.